onnxruntime/python/tools/transformers/run_benchmark.sh

```diff
@@ -0,0 +1,142 @@
+# This measures the performance of OnnxRuntime, PyTorch and TorchScript on transformer models.
+# Please install PyTorch (see https://pytorch.org/) before running this benchmark. Like the following:
+# GPU:   conda install pytorch torchvision cudatoolkit=10.1 -c pytorch
+# CPU:   conda install pytorch torchvision cpuonly -c pytorch
+
+# When run_cli=true, this script is self-contained and you need not copy other files to run benchmarks
+#                    it will use onnxruntime-tools package.
+# If run_cli=false, it depends on other python script (*.py) files in this directory.
+run_cli=true
+
+# only need once
+run_install=true
```

Comment:
Why are "run_cli" and "run_install" variables default to true here but default to false in the other .cmd file?

Reply:
dev_benchmark is used for ORT developer. So it assumes the default will be different:

(1) dev will test the script in master branch instead of  cli in the released onnxruntime_tools package.
(2) dev usually has installed related packages (only need once), so default is false.